src/devices/Ina226/samples/README.md

```diff
@@ -0,0 +1,124 @@
+﻿# Ina226 Sample Project
```

Comment:
The samples should no longer have their own readme's. This should be included in the readme of the parent folder (we have build an automated process that includes it in the API documentation from there).

(Note to self: We should update the template)

Reply:
I will get these two combined into the regular ina226 read me.